iOS_SDK/OneSignalSDK/Source/OneSignalNotificationData.h

```diff
@@ -1,38 +0,0 @@
-/**
- * Modified MIT License
- *
- * Copyright 2017 OneSignal
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * 1. The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * 2. All copies of substantial portions of the Software may only be used in connection
- * with services provided by OneSignal.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-
-#import "OSLastNotification.h"
-
-@interface OneSignalNotificationData : NSObject
-
-+ (void)saveLastNotificationFromBackground:(NSString * _Nullable)notificationId;
-+ (void)saveLastNotification:(NSString * _Nullable)notificationId;
-+ (void)saveLastNotificationWithBackground:(NSString *  _Nullable)notificationId wasOnBackground:(BOOL)wasOnBackground;
-+ (OSLastNotification * _Nonnull)getLastNotification;
-+ (NSString * _Nullable)getLastNotificationId;
-
-@end
```

Comment:
Why are we removing these unnecessary `.h` and `.m` files in this PR?

Reply:
They prevent the project from compiling when using SwiftPM so we need to remove them for this to work.